crates/leon/Cargo.toml

```
@@ -0,0 +1,27 @@
+[package]
+name = "leon"
+description = "Dead-simple string templating"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+documentation = "https://docs.rs/leon"
+version = "0.0.0"
+rust-version = "1.61.0"
+authors = ["Félix Saparelli <felix@passcod.name>"]
+edition = "2021"
+license = "Apache-2.0 OR MIT"
+
+[dependencies]
+miette = "5.5.0"
+thiserror = "1.0.38"
+
+[dev-dependencies]
+criterion = "0.4.0"
```

```suggestion
criterion = { version = "0.4.0", default-features = false, features = ["cargo_bench_support"] }
```

criterion pulls in too much dependencies we don't need, I would like to reduce amount of changes in `Cargo.lock`.

Also, since we are using nightly, perhaps we could turn on feature `real_blackbox` for more robust result?